client/my-sites/stats/post-performance/style.scss

```diff
@@ -0,0 +1,7 @@
+.stats-post-performance__start-post {
+	padding-bottom: 26px;
+}
+
+.stats-post-performance__no-posts-message {
+	margin-bottom: 1em;
```

Comment:
Well, you could retain `em` if need be, but I wasn't sure why it's two different units instead of just using one or the other.